sql/builtin_functions.cc

```diff
@@ -3141,6 +3141,35 @@ BoundFunction hash(const std::vector<BoundSqlExpression> & args)
 
 static RegisterBuiltin registerHashFunction(hash, "hash");
 
+BoundFunction tryFct(const std::vector<BoundSqlExpression> & args)
+{
+    checkArgsSize(args.size(), 2);
+    auto outputInfo
+        = std::make_shared<UnknownRowValueInfo>();
+    auto bindBuiltinFunction = [=] (SqlBindingScope & scope,
+                            std::vector<BoundSqlExpression>& boundArgs,
+                            const SqlExpression * expr) ->  BoundSqlExpression
+    {
+        return {[=] (const SqlRowScope & row,
+                     ExpressionValue & storage,
+                     const VariableFilter & filter) -> const ExpressionValue &
+        {
+            ExcAssertEqual(boundArgs.size(), 2);
+            try {
+                return storage = std::move(boundArgs[0](row, GET_LATEST));
+            }
+            catch (const HttpReturnException & exc) {
```

Comment:
We should use JML_CATCH_ALL if we don't care about the type of the exception, or at least std::exception.